src/main/java/org/jabref/gui/JabRefFrame.java

```diff
@@ -213,6 +208,14 @@ private void init() {
         Pane containerPane = DndTabPaneFactory.createDefaultDnDPane(DndTabPaneFactory.FeedbackType.MARKER, null);
         tabbedPane = (DndTabPane) containerPane.getChildren().get(0);
 
+        uiUpdate = new Observable() {
```

Comment:
this looks a bit odd. Why don't you use a ChangeListener or sth like that?

Reply:
I decided to go with the simple Observable object because of the way everything is laid out. I could create an Observable Map and apply a change listener to that so when the button changes it can update the HBox.